Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.7-slim-buster AS base
+EXPOSE 5000
+WORKDIR /todo-app
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+ENV PATH="${PATH}:/root/.poetry/bin"
+COPY ./poetry.toml ./pyproject.toml ./
```

Comment:
You ought to copy in the poetry.lock file to make sure you use the exact same versions of all your dependencies